cypress/integration/pages/onDemandRadio/tests.js

```diff
@@ -43,7 +43,7 @@ export default ({ service, pageType, variant, isAmp }) => {
             const iframeURL = isBrandPage ? iframe.prop('src') : embedUrl;
 
             cy.get(`iframe[src*="${iframeURL}"]`).should('be.visible');
-            cy.testResponseCodeAndType(iframeURL, 200, 'text/html');
+            cy.testResponseCodeAndTypeRetry(iframeURL, 200, 'text/html');
```

Comment:
Sorry for lurking on this PR 🙈 - would it also be possible to add this to the other tests which check the status of the media player - live radio, MAPs & articles? Just to improve the overall stability of the E2Es?

Reply:
It was my plan to do that eventually, but I wanted to test it with these two page types first to see if it works! 
(1) It works on my laptop, but I wanted to make sure it didn't do anything weird when in the pipeline. 
(2) I don't know if it is actually solving the problem or if I just got lucky with my test runs having no flakes. 

However, if from looking at it you think it looks pretty safe, we can add it to other page types too.

Reply:
Totally up to you. I think the others would be just fine, but if you're more comfortable to only do On Demand TV & Radio first to see if it improves, then that makes sense.